src/olympia/reviewers/tests/test_views.py

```diff
@@ -6020,6 +6020,76 @@ def test_abuse_reports_resolved_as_approve_with_approve_latest_version_action(
         assert decision.action == DECISION_ACTIONS.AMO_APPROVE_VERSION
         self.assertCloseToNow(decision.action_date)
 
+    def test_force_enable_shows_versions_to_reenabled(self):
+        v1 = self.addon.current_version
+        v2 = version_factory(addon=self.addon)
+        v3 = version_factory(addon=self.addon)
+        v4 = version_factory(addon=self.addon, channel=amo.CHANNEL_UNLISTED)
+        self.addon.update(status=amo.STATUS_DISABLED)
+        v1.file.update(
+            status=amo.STATUS_DISABLED,
+            original_status=amo.STATUS_APPROVED,
+            # We don't want to re-enable a version the developer disabled
+            status_disabled_reason=File.STATUS_DISABLED_REASONS.DEVELOPER,
+        )
+        v2.file.update(
+            status=amo.STATUS_DISABLED,
+            original_status=amo.STATUS_APPROVED,
+            # we also don't want to re-enable a version we rejected
+            status_disabled_reason=File.STATUS_DISABLED_REASONS.NONE,
+        )
+        v3.file.update(
+            status=amo.STATUS_DISABLED,
+            original_status=amo.STATUS_APPROVED,
+            # but we do want to re-enable a version we only disabled with the addon
+            status_disabled_reason=File.STATUS_DISABLED_REASONS.ADDON_DISABLE,
+        )
+        v4.file.update(
+            status=amo.STATUS_DISABLED,
+            original_status=amo.STATUS_AWAITING_REVIEW,
+            # awaiting review versions could also be re-enabled
+            status_disabled_reason=File.STATUS_DISABLED_REASONS.ADDON_DISABLE,
+        )
+
+        response = self.client.get(self.url)
+        self.assertContains(response, '2 version(s) will be re-enabled:')
+        doc = pq(response.content)
+        items = doc('li.version-that-would-be-reenabled')
+        assert len(items) == 2
+        assert v3.version in items[1].text_content()
+        assert '-> Approved' in items[1].text_content()
+        assert 'Listed' in items[1].text_content()
+        assert v4.version in items[0].text_content()
+        assert '-> Awaiting Review' in items[0].text_content()
+        assert 'Unlisted' in items[0].text_content()
+        assert len(doc('li.version-that-would-be-reenabled.overflow-message')) == 0
+        assert doc('li.version-that-would-be-reenabled a').eq(1).attr.href == reverse(
+            'reviewers.review_version_redirect', args=[self.addon.id, v3.version]
+        )
+        assert doc('li.version-that-would-be-reenabled a').eq(0).attr.href == reverse(
+            'reviewers.review_version_redirect', args=[self.addon.id, v4.version]
+        )
+
+    def test_force_enable_shows_versions_to_reenabled_too_many(self):
```

Comment:
12 seems arbitrary, can you override the setting you're using to determine if there are "too many" so it is 1) clear in the test why that number is relevant and 2) the test is immune to changing the value.

Reply:
12 was just so the test for 11 rows (10 plus the overflow row) was unambigious, but I can re-work...

Reply:
added comments and moved the template constant up to a real constant, so it can be overridden.